bo/technischeressource.go

```diff
@@ -0,0 +1,32 @@
+	VorgelagerteMesslokationsId      string                                                            `json:"vorgelagerteMesslokationsId,omitempty" example:"DE00713739359S0000000000001222221" validate:"alphanum,required,len=33"` // Vorgelagerte Messlokation ID
+	ZugeordneteMarktlokationsId      string                                                            `json:"zugeordneteMarktlokationsId,omitempty" example:"20072281644"`                                                           // ZugeordneteMarktlokationsId Messlokation ID
+	ZugeordneteSteuerbareRessourceId string                                                            `json:"zugeordneteSteuerbareRessourceId" example:"20072281644"`                                                                // Referenz auf die der Technischen Ressource zugeordneten Steuerbaren Ressource
+	NennleistungAufnahme             com.Menge                                                         `json:"nennleistungAufnahme" example:"QTY+Z43:100:KWT"`                                                                        // Nennleistung (Aufnahme)
+	NennleistungAbgabe               com.Menge                                                         `json:"nennleistungAbgabe" example:"QTY+Z44:100:KWT"`                                                                          //Nennleistung (Abgabe)
+	Speicherkapazitaet               com.Menge                                                         `json:"speicherkapazitaet" example:"QTY+Z42:100:KWH"`                                                                          //Speicherkapazität
+	TechnischeRessourceNutzung       technischeressourcenutzung.TechnischeRessourceNutzung             `json:"technischeressourcenutzung" example:"CCI+Z17"`                                                                          //Art und Nutzung der Technischen Ressource
+	Verbrauchsart                    technischeressourceverbrauchsart.TechnischeRessourceVerbrauchsart `json:"verbrauchsart" example:"CAV+Z64" `                                                                                      //Verbrauchsart der Technischen Ressource
+	Waermenutzung                    waermenutzung.Waermenutzung                                       `json:"waermenutzung" example:"CAV+Z56"`                                                                                       //Wärmenutzung
+	EMobilitaetsart                  emobilitaetsart.EMobilitaetsart                                   `json:"eMobilitaetsart" example:"CAV+Z87"`                                                                                     //Art der E-Mobilität  Das Segment dient dazu, im Falle der E-Mobilität eine genauere Angabe über die Art der E-Mobilität zu definieren
```

Comment:
casing:
https://github.com/Hochfrequenz/BO4E-dotnet/blob/efe49e5d6ed879cc9e0637225812276c512d1d54/BO4E/BO/TechnischeRessource.cs#L154


```suggestion
	EMobilitaetsart                  emobilitaetsart.EMobilitaetsart                                   `json:"emobilitaetsart" example:"CAV+Z87"`                                                                                     //Art der E-Mobilität  Das Segment dient dazu, im Falle der E-Mobilität eine genauere Angabe über die Art der E-Mobilität zu definieren
```